### PR TITLE
Use Jekyll for the website

### PR DESCRIPTION
**Pull request by xtuc**
Preparation for #12.

This can't be merged in gh-pages, could you create a master branch?

### CODE REVIEW

**Comment by hzoo** [comment]
Cool, changed master, just need to fix merge conflicts now

(and maybe the users part can be moved to another file later)

File: _layouts/compress.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (jamiebuilds) [inline _layouts/compress.html]: We don't need this, we can just put it behind CloudFlare which can minify everything for us

Comment (hzoo) [reply _layouts/compress.html]: Nice! were doing the same in Babel right now I think

Comment (xtuc) [reply _layouts/compress.html]: Ok, that's cool. I've updated the PR.

**Review by doug-wade — APPROVED** [review]

**Comment by JamesHenry** [comment]
Hi @xtuc 👋 

Thanks a lot for taking the initiative to contribute this PR!

You may or may not know that lerna is now under the stewardship of Nrwl (announcement here https://github.com/lerna/lerna/issues/3121), a company with a long history of not just producing valuable open-source software (OSS), but also backing others (at the time of writing, Nrwl has donated over $50,000 to OSS it _hasn't_ created, see https://opencollective.com/nx for full details).

Quite simply, Nrwl ❤️  OSS, and is committed to making lerna the best it can be. We use it ourselves.

We hope you will continue to be a part of this community as we look to take things forward from here!

Please see https://github.com/lerna/lerna/discussions/3140 for more details on our plans for 2022.

We have just pushed out a new version of the website, which we will continue to iterate on, we hope you like it and find it useful!

Feel free to open a new issue on here if you spot anything that needs fixing or improving, but for now this PR is no longer applicable to the current site so we are closing it.

Many thanks 🙏

**Pull request closed without merging**